tests/ga/test_extension.py

```diff
@@ -3425,49 +3426,6 @@ def mock_http_put(url, *args, **_):
 
             self.assertEqual(expected_status, actual_status_json)
 
-    def test_it_should_zip_waagent_status_when_incarnation_changes(self):
```

Comment:
Now we save directly to the history folder so this behavior does not exist anymore